modules/mobile/config/routes.rb

```diff
@@ -86,5 +86,13 @@
   namespace :v1 do
     get '/health/immunizations', to: 'immunizations#index'
     get '/user', to: 'users#show'
+
+    scope :messaging do
+      scope :health do
+        resources :messages, only: %i[], defaults: { format: :json } do
```

Comment:
Is the `only` array empty as to not create standard restful actions for this path since it's using a custom action?

Reply:
I also don't really get this. I tried removing the `only` and it gives a whole resource full of routes for messages. So it's definitely necessary. I suspect there's another way of structuring this to avoid the only, but this does seem to work.

Reply:
Rails routes default to producing all resourceful routes for the resource provided. You override with either `only` or `except`. Obviously `except` would be way more wordy, so `only` is best. It's a bit weird, but it's necessary if you don't want to manually specify every `get`